builder/oracle/classic/step_add_keys.go

```diff
@@ -50,6 +56,10 @@ func (s *stepAddKeysToAPI) Run(_ context.Context, state multistep.StateBag) mult
 func (s *stepAddKeysToAPI) Cleanup(state multistep.StateBag) {
 	// Delete the keys we created during this run
 	keyName := state.Get("key_name").(string)
+	if keyName == "" {
```

Comment:
IMO this would be more idiomatic, but how it is now is fine

```go
keyName, ok := state.Get("key_name")
if !ok {
    return
}
// ...
deleteInput := compute.DeleteSSHKeyInput{Name: keyName}
```